src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Scaffolders/Scaffolder.cs

```diff
@@ -42,7 +41,11 @@ public async Task ExecuteAsync(ScaffolderContext context)
             preparer.RunPreExecute(step, context);
             if (!step.SkipStep)
             {
-                await step.ExecuteAsync(context);
+                var stepResult = await step.ExecuteAsync(context);
+                if (!stepResult && !step.ContinueOnError)
+                {
+                    break;
+                }
             }
 
             preparer.RunPostExecute(step, context);
```

Comment:
Probably a discussion we should have at some point about whether the PostExecute should run (with a failed flag?) even if there was an error. Not for now though

Reply:
ok added a TODO for now